src/celutil/utf8.h

```diff
@@ -114,12 +115,17 @@ class Greek
     };
 
     static const std::string& canonicalAbbreviation(const std::string&);
-
+private:
+    static Greek* m_instance;
  public:
-    static Greek* instance;
+    static Greek* getInstance();
     int nLetters;
     std::string* names;
     std::string* abbrevs;
 };
 
+int findGreekNameIndexBySubstr(const std::string &, int = 0, uint = 0xffffffff);
+std::string firstGreekAbbrCompletion(const std::string &);
+std::vector<std::string> getGreekCompletion(const std::string &);
+
```

Comment:
Maybe these 3 functions better to make private of utf8.cpp? (move prototypes into cpp file and add `static`)